packages/cli/src/commands/lint.ts

```diff
@@ -108,3 +123,53 @@ export async function handleLint(
     process.exit(totals.errors === 0 || argv['generate-ignore-file'] ? 0 : 1),
   );
 }
+
+function lintConfigCallback(argv: LintOptions, version: string) {
+  if (argv['lint-config'] === 'off') {
+    return;
+  }
+
+  return async (config: RawConfig) => {
+    const configPath = findConfig(argv.config) || '';
+    const stringYaml = stringifyYaml(config);
+    const configContent = makeDocumentFromString(stringYaml, configPath);
+    const problems = await lintConfig({
+      document: configContent,
+      severity: argv['lint-config'] as ProblemSeverity,
+    });
+    await validateLintConfig({
+      problems,
+      outputFormat: argv.format,
+      maxProblems: argv['max-problems'],
+      version,
+    });
+  };
+}
+
+async function validateLintConfig({
+  problems,
+  outputFormat,
+  maxProblems,
+  version,
+}: {
+  problems: NormalizedProblem[];
+  outputFormat: OutputFormat;
+  maxProblems?: number;
+  version: string;
+}): Promise<void> {
+  const fileTotals = getTotals(problems);
+
+  formatProblems(problems, {
+    format: outputFormat,
+    maxProblems,
+    totals: fileTotals,
+    version,
+  });
+  printConfigLintTotals(fileTotals);
+}
+
+function printValidatingFile(filePath: string, validationDuration: number | string): void {
+  process.stderr.write(
+    gray(`${relativeFilePath(filePath)}: validated in ${validationDuration}\n\n`),
```

Comment:
is `validationDuration` string or number? if it's a number do we need to add `ms` unit?

```suggestion
    gray(`${relativeFilePath(filePath)}: validated in ${validationDuration}ms\n\n`),
```

Reply:
As argument `validationDuration = getExecutionTime(startedAt)` and already return it with `ms` but there is no need for number type in such case. Due to removing `relativeFilePath` there is no more need in this fn.